### PR TITLE
Adding the ROBIN proposal

### PR DESCRIPTION
**Pull request by suprajasekhar**
This proposal aims to conceptualize how ad quality checks and publisher controls enforcement can be supported in TURTLEDOVE for RTB demand.

### CODE REVIEW

**Review by sbelov — CHANGES_REQUESTED** [review]
Thank you – a couple of formatting nits.

File: proposals/robin/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+Upon receiving an interest group request, a DSP can request interest group creative metadata from their SSP for the list of creatives they intend to use. If the DSP’s creative does not require pre-approval, the SSP can instantly return its signed approval status. The SSP can then scan the creative for ad quality compliance and classifications needed for enforcing publisher controls. In the subsequent interest group creative metadata request, the SSP can provide the signed metadata detected for the creative.
+
+Considering a DSP often partners with several SSPs, it may be beneficial to standardize the protocol for sharing the interest group creatives and their associated metadata between DSPs and SSPs. Interest group creative metadata request from a DSP to an SSP can contain an object for each creative and may look as follows:
+
+```
+[{      'creativeId': 'ad123'
```
[the remark this answers is not in the document]
Comment (suprajasekhar) [reply proposals/robin/README.md]: Done.

File: proposals/robin/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+Optionally, SSPs can provide a creative scanning API that allows DSPs to submit creatives for scanning and verification prior to receiving interest group requests. This approach can reduce interest group ad filtering when working with SSPs that require creatives to be scanned before the first impression. 
+
+## On-device Policy and Publisher Controls Enforcement
+![robin policy and publisher controls enforcement](./robin-policy-and-publisher-controls-enforcement.png)
```
[the remark this answers is not in the document]
Comment (suprajasekhar) [reply proposals/robin/README.md]: Sure!

**Comment by suprajasekhar** [comment]
Thank you! Updated based on review comments.

**Pull request merged**